registration/tests/test_admin.py

```diff
@@ -551,31 +550,31 @@ def test_save_model(self):
         cash_drawer.delete()
 
 
-class TestTwoFactorAdmin(TestCase):
-    def setUp(self):
-        self.user_profile_admin = admin.UserProfileAdmin(
-            model=User, admin_site=AdminSite()
-        )
-        self.user_1 = User.objects.create_user("john", "lennon@thebeatles.com", "john")
-        self.user_2 = User.objects.create_superuser("admin", "admin@host", "admin")
-        self.user_1.staff_member = True
-        self.user_1.save()
-        self.user_2.save()
-
-    def test_two_factor_disabled(self):
-        self.assertFalse(self.user_profile_admin.two_factor_enabled(self.user_1))
-
-    def test_two_factor_enabled(self):
-        self.user_2.u2f_keys.create(
-            key_handle="bbavVvfXPz2w8S3IwIS0LkE1SkC3MQuXSYjAYHVPFqUJIRQTIEyM3D34Lv2G4a_PuAZkZIQ6XV3ocwp47cPYjg",
-            public_key="BFp3EHDcpm5HxA4XYuCKlnNPZ3tphVzRvXsX2_J33REPU0bgFgWsUoyZHz6RGxdA84VgxDNI4lvUudr7JGmFdDk",
-            app_id="http://localhost:8000",
-        )
-        self.assertTrue(self.user_profile_admin.two_factor_enabled(self.user_2))
-
-    def test_disable_two_factor(self):
-        query_set = [self.user_1, self.user_2]
-        admin.disable_two_factor(None, None, query_set)
+# class TestTwoFactorAdmin(TestCase):
```

Comment:
It might be possible to upgrade this library to support DJango 5, but might be wiser to switch to something more actively developed like [django-allauth](https://docs.allauth.org/en/latest/).

Reply:
I will look into it, and re-add 2FA.